### DISCUSSION TITLE
How to simultaneously edit parameters in a parenthesis using meow-grab?

### DISCUSSION THREAD

**Discussion by K-gihu** [meow-edit/meow#243] [Q&A] [answered]
I want to change `[65, 130, 325, 650]` to `["65", "130", "325", "650"]` in the following line:
```python
    for sidelen in [65, 130, 325, 650]:
```
I tried to select the line and grab it, but if I `meow-find` the spaces, the spaces before the list will be edited. If  I `meow-back-word` to make fake cursor, still cursors will be made on previous words. Is there a solution?

**Comment by DogLooksGood** [comment, marked as answer]
How about grab the list, not the whole line.

**Reply by K-gihu** [reply]
thanks, forgot about it.
